### PR TITLE
Added white border to interstate shields.

### PR DESCRIPTION
**Pull request by amcginni**
This pull request adds a white border around the interstate shield, consistent with the MUTCD and other online maps. 

Before:
image

After:
image

### CODE REVIEW

**Comment by amcginni** [comment]
Based on feedback in [Slack](https://osmus.slack.com/archives/C01V02K52UX/p1643346755900879l), the thickness of the borders was increased, and the padding was adjusted to vertically center the text.  Below are some sample renderings of the changes.
image
image

File: style/icons/shield40_us_interstate_2.svg [hunk 1]
```diff
@@ -5,38 +5,60 @@
    version="1.0"
```
Comment (ZeLonewolf) [inline style/icons/shield40_us_interstate_2.svg:5]: Please save the SVGs as "optimized SVG" in Inkscape to save on space.

File: style/package-lock.json [hunk 2]
```diff
@@ -1,3169 +1,6 @@
 {
```
Comment (ZeLonewolf) [inline style/package-lock.json]: This appears to be an unrelated change.

File: style/icons/shield40_us_interstate_2.svg (hunk 1 above)
Comment (amcginni) [reply style/icons/shield40_us_interstate_2.svg]: This is addressed in efa8f21f35f3d01a68bcf3d04cbda7553f26e7d5

**Comment by ZeLonewolf** [comment]
Code looks good to me.  I'll leave it to the style gurus to comment on the graphics.

**Comment by zekefarwell** [comment]
The addition of the white lines looks good.  I think they are currently too thick though.  The white removes a fair about of blue and red from the overall shield, and ends up looking a bit chunky.  In these screenshots I've tweaked `shield40_us_interstate_2.svg` to make the lines thinner, but left `shield40_us_interstate_3.svg` the same to illustrate the difference.  With thinner lines I think this will look excellent.

at 1x
Screen Shot 2022-01-30 at 10 52 01 PM

at 2x
Screen Shot 2022-01-30 at 10 53 06 PM

**Review by zekefarwell — CHANGES_REQUESTED** [review]
The addition of the white lines looks good.  I think they are currently too thick though.  The white removes a fair about of blue and red from the overall shield, and ends up looking a bit chunky.  In these screenshots I've tweaked `shield40_us_interstate_2.svg` to make the lines thinner, but left `shield40_us_interstate_3.svg` the same to illustrate the difference.  With thinner lines I think this will look excellent.

at 1x
Screen Shot 2022-01-30 at 10 52 01 PM

at 2x
Screen Shot 2022-01-30 at 10 53 06 PM

**Comment by amcginni** [comment]
I agree, the original borders are too thick.  This is addressed in 2d13f819e3b1dbcd06d4d67820f00af83927bf01.  Below is a rendering of two and three digit shields, where the border around the shield is clearly visible.
image

**Review by zekefarwell — APPROVED** [review]
Looks good to me.  Thanks!
